Update privileged-access-deployment.md

mjcaparas opened the pull request:
Removing reference to the expedite mode. See for reference: https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/optimized-reporting-latency-and-expedite-mode/ba-p/229208

opbld31 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[security-docs/Solving-TLS1-Problem.md](https://github.com/v-savila/Security/blob/task52641_last/security-docs/Solving-TLS1-Problem.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/security/solving-tls1-problem?branch=pr-en-us-17) |
[security-docs/index.md](https://github.com/v-savila/Security/blob/task52641_last/security-docs/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/security/index?branch=pr-en-us-17) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C6%5C785696d2-000e-9124-f54d-f7fc229d428a%5CPullRequest%5C201812060142348452-17%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tXaNXNzRj6cJnrz2dYnkwYtMFrshFzGD3xNyt9EjWeM%3D&st=2018-12-06T01%3A38%3A32Z&se=2019-01-06T01%3A43%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

opbld32 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[security-docs/Solving-TLS1-Problem.md](https://github.com/v-savila/Security/blob/task52641_last/security-docs/Solving-TLS1-Problem.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/security/solving-tls1-problem?branch=pr-en-us-17) |
[security-docs/index.md](https://github.com/v-savila/Security/blob/task52641_last/security-docs/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/security/index?branch=pr-en-us-17) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C6%5C785696d2-000e-9124-f54d-f7fc229d428a%5CPullRequest%5C201812060148455356-17%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wVIcmx3dKfzJqkmYUQepzMHNEc%2FkAEmxo%2BdQUvN92ng%3D&st=2018-12-06T01%3A44%3A41Z&se=2019-01-06T01%3A49%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

mjcaparas committed:
Update privileged-access-deployment.md

Removing reference to the expedite mode. See for reference: https://techcommunity.microsoft.com/t5/microsoft-defender-for-endpoint/optimized-reporting-latency-and-expedite-mode/ba-p/229208
